### PR TITLE
KAFKA-3920: Add Schema source connector to Kafka Connect

### PR DESCRIPTION
**Pull request by Ishiihara**

### CODE REVIEW

**Comment by Ishiihara** [comment]
@ewencp Ping for review.

File: connect/runtime/src/main/java/org/apache/kafka/connect/tools/SchemaSourceTask.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    private String topic;
+    private Map<String, Integer> partition;
+    private long startingSeqno;
+    private long seqno;
+    private long count;
+    private long maxNumMsgs;
+    private boolean multipleSchema;
+    private int partitionCount;
+
+    // Until we can use ThroughputThrottler from Kafka, use a fixed sleep interval. This isn't perfect, but close enough
```
Comment (ewencp) [inline connect/runtime/src/main/java/org/apache/kafka/connect/tools/SchemaSourceTask.java]: We have ThroughputThrottler here, should we just fix this to use it?

**Comment by ewencp** [comment]
@Ishiihara I think you want to update ConnectorPluginsResourceTest.java to validate these don't leak out in the list connectors API.

**Comment by Ishiihara** [comment]
@ewencp Thanks for the review. Updated the PR to address comments. PTAL.

**Comment by ewencp** [comment]
LGTM

**Pull request closed without merging**